src/games/strategy/triplea/ui/TripleaMenu.java

```diff
@@ -410,10 +403,37 @@ public void actionPerformed(final ActionEvent e) {
     });
     menuGame.add(mapZoom).setMnemonic(KeyEvent.VK_Z);
   }
+  
+  private void addUnitNationDrawMenu(final JMenu parentMenu){
+    final JMenu unitSizeMenu = new JMenu();
+    unitSizeMenu.setMnemonic(KeyEvent.VK_N);
+    unitSizeMenu.setText("Show Unit Nation");
+    final ButtonGroup unitNationGroup = new ButtonGroup();
+    final JRadioButtonMenuItem nationOption0 = new JRadioButtonMenuItem("Hide Nation");
+    final JRadioButtonMenuItem nationOption1 = new JRadioButtonMenuItem("Next to Unit");
+    final JRadioButtonMenuItem nationOption2 = new JRadioButtonMenuItem("Below Unit");
+    nationOption0.addActionListener(SwingAction.of(e -> {
+      UnitsDrawer.setNationVisible((byte)0);
+      frame.getMapPanel().resetMap();
+    }));
+    nationOption1.addActionListener(SwingAction.of(e -> {
+      UnitsDrawer.setNationVisible((byte)1);
+      frame.getMapPanel().resetMap();
+    }));
+    nationOption2.addActionListener(SwingAction.of(e -> {
+      UnitsDrawer.setNationVisible((byte)2);
+      frame.getMapPanel().resetMap();
+    }));
+    nationOption2.setSelected(true);
```

Comment:
Here is where you would want to make the option 'sticky', and have the chosen option be retained between game restarts. The other menu settings should have some examples.

To be explicit, around here you would load the user selected option, defaulting to something if not set, then when changed you would update that setting.